03_module/hello.c

```diff
@@ -0,0 +1,19 @@
+#include <linux/module.h>
```

Comment:
WARNING: Missing or malformed SPDX-License-Identifier tag in line 1
#1: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/hello.c:1:
+#include <linux/module.h>